bin/ghe-backup

```diff
@@ -1,12 +1,42 @@
 #!/usr/bin/env bash
-#/ Usage: ghe-backup [-v] [--version]
+#/ Usage: ghe-backup [-hv] [--version]
+#/
 #/ Take snapshots of all GitHub Enterprise data, including Git repository data,
 #/ the MySQL database, instance settings, GitHub Pages data, etc.
 #/
-#/ With -v, enable verbose output and show more information about what's being
-#/ transferred.
+#/ OPTIONS:
+#/   -v | --verbose    Enable verbose output
+#/   -h | --help       Show this message
+#/        --version    Display version information
```

Comment:
Nit: need full stops at the end of each of these to be consistent with elsewhere.